_sass/base/_markdown.scss

```diff
@@ -26,13 +26,14 @@
   // List Items
   li {
     line-height: 24px;
+    margin: 5px;
   }
 
   // Lists
   ol,
   ul {
     list-style: initial;
-    font-size: 15px;
+    font-size: 16px;
```

Comment:
And anyway, why do we need to override font-size and font-weight for lists? Shouldn't those just be the same as the main text of the doc?